WordPress/Classes/ViewRelated/Reader/ReaderInterestsCoordinator.swift

```diff
@@ -34,6 +34,19 @@ class ReaderSelectInterestsCoordinator {
         }()
     }
 
+    // MARK: - Saving
+    public func saveInterests(interests: [RemoteReaderInterest], completion: @escaping (Bool) -> Void) {
+        let isLoggedIn = userId != nil ? true : false
```

Comment:
Also, why were we using `ReaderHelpers.isLoggedIn()` till now but now we're checking it like this?

Reply:
Both do the same.

I'd say that the advantage here is that since we have the `userId` we can rely directly on this property instead of calling `ReaderHelpers.isLoggedIn()  — which will rely on Core Data again.

Anyway, both will work. Do you prefer to change? Is that confusing to you?